src/styles/SearchForm.css

```diff
@@ -0,0 +1,49 @@
+body {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+.wrapper {
+  width: 100%;
+  max-width: 31.25rem;
+  margin: 6rem auto;
+}
+
+.label {
+  font-size: 0.625rem;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: +1.3px;
+  margin-bottom: 1rem;
+}
+
+.search-form {
+  width: 95%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: auto;
+}
+
+#searchQueryInput {
```

Comment:
Could this not be a class instead? For styling we should avoid using ids as they are extremely specific and hard to override should a child class in the future need to take precedence